lib/blockstore/common.js

```diff
@@ -0,0 +1,31 @@
+/*!
+ * common.js - blockstore constants for bcoin
+ * Copyright (c) 2019, Braydon Fuller (MIT License).
+ * https://github.com/bcoin-org/bcoin
+ */
+
+'use strict';
+
+/**
+ * @module blockstore/common
+ */
+
+/**
+ * Block data types.
+ * @enum {Number}
+ */
+
+exports.types = {
+  BLOCK: 1,
+  UNDO: 2
+};
+
+/**
+ * File prefixes for block data types.
+ * @enum {String}
+ */
+
+exports.prefixes = {
+  1: 'blk',
+  2: 'blu'
```

Comment:
I noticed Core prexies undo blocks with `rev` -- wonder if we should match that now so compatibility is easier later

Reply:
...ah although I see the undo blocks dont matter when importing data from Core, so I guess it doesn't matter

Reply:
Yeah, those are not compatible. From reading the code there are some small differences, and thus there is a different prefix to avoid any mistakes. The bcoin format has some additional changes to include the blockhash in the prefix magic so that the indexes can be regenerated from the files directly.